Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py

```diff
@@ -800,6 +805,82 @@ def update_remote_system_command(client: Client, args: Dict[str, Any], mirror_ta
     return new_incident_id
 
 
+def are_two_datetime_equal_by_second(x: datetime, y: datetime):
+    """Calculate if two datetime objects are equal up to the seconds value.
+        Even though the 'time' attribute of each event has milliseconds,
+        the API request supports time filtering of only up to seconds.
+        There for, all events with the same time up to a seconds are considered to have the same time.
+
+    Args:
+        x (datetime): First datetime.
+        y (datetime): Second datetime.
+
+    Returns:
+        Boolean: True if both datetime objects have the same time up to seconds, False otherwise.
+    """
+    return (x.year == y.year) and (x.month == y.month) and (x.day == y.day)\
```

Comment:
can be replaced with `x.replace(microsecond=0)` and the same for y and simply compare them.

Reply:
We have milliseconds in XSOAR6, so this comparison will be wrong in XSOAR6. Therefore, I only considered seconds and not milliseconds. Please explain your comment. I think I don't understand your meaning